python/cugraph/cugraph/centrality/plc_betweenness_centrality.py

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) 2019-2022, NVIDIA CORPORATION.
```

Comment:
I don't like the name "plc_betweenness_centrality".  That makes it sound like it's a part of `pylibcugraph` when it's intended to be a replacement for the existing betweenness centrality.  Could you give it a better name, like "fast_betweenness_centrality"?

Reply:
Also, this file appears to be missing tests.